README.md

```diff
@@ -1,6 +1,16 @@
 Olin.js - Spring 2015
 ===
 
+##Course Description
+Olin.js is a project oriented approach to learning modern web application development. With server and client technology advancing so rapidly, the modern website looks very different from that of even a few years ago, and the web will only become a bigger part of our lives in the near future. Olin.js will quickly familiarize students with Node.js as a web server framework and ES5 JavaScript as a responsive client-side language with three weeks of introductory instruction and four weeks of partner labs.
+
+Along the way, students will learn the basic layout of the web, how to deploy and maintain large applications, important development strategies, skills, and technologies, and how to design and manage complex, multi-developer software projects. Students will spend the last six weeks of the course putting their knowledge and skills to the test by designing and developing their own web applications.
+
+###Learn Web Development
+If you stumbled across this repository and feel like learning some web development please follow along with us.
```

Comment:
I'd change to:

If you stumbled across this repository and feel like learning some web development, please follow along with us!